moped-editor/src/views/projects/projectView/NoteInputQuill.js

```diff
@@ -78,25 +99,95 @@ const NoteInputQuill = ({
   isStatusEditModal,
 }) => {
   const classes = useStyles();
-  const ref = useRef();
 
-  useEffect(() => {
-    // autofocuses the quill input
-    ref?.current.focus();
-  }, []);
+  const initialConfig = {
+    namespace: 'MyEditor',
+    theme,
+    nodes: [
+      LinkNode,
+    ],
+    onError,
+  };
+
+  // On change, return editor content as HTML
+  const OnChangePlugin = ({ onChange }) => {
```

Comment:
@tillyw I was taking a preview of this code to see how this works, and I noticed that these plugins are components in themselves. Could you move their definitions outside the `NoteInputQuill` component. You'll probably have to pass some props.

This is cool to see! The MUI DataGrid Pro work that Chia did recently used a similar pattern with a table context that was used in child components.

Reply:
thanks, @mddilley! i just pushed an update that makes these fixes and cuts out some unnecessary code. will request review shortly!